### ISSUE TITLE
Feature: Adding extra text to be share with QR code in android sharesheet in showQrActivity

### ISSUE THREAD

**Issue by Harshad09** [openMF/mobile-wallet#1131]
**Summary:**

Adding extra text to be shared with QR code with Intent in android Sharesheet in showQrActivity, for better user experience.

**Steps to reproduce:**

1. Go to Request fragment in Payments 
2. Click on show code
3. Click on the share icon (located at top right)
4. Choose any app to share

**Expected behavior:**

The extra text should be share with a QR code to ease the user experience

**Device and Android version:**

Redmi Note 5 Pro
Android 9 Pie
custom ROM

**Comment by devansh-299** [comment]
Not sure if we have the deep link handled for `Scan QR` functionality but in case we have the feature, it would be better to pass the deep link in the text as well.

**Comment by Harshad09** [comment]
> Not sure if we have the deep link handled for `Scan QR` functionality but in case we have the feature, it would be better to pass the deep link in the text as well.

Yes, Deep linking functionality should be there, as for now is this feature acceptable or I have some suggested changes @devansh-299

**Comment by Harshad09** [comment]
PhonePe also uses this type of extra text when we share qr code.

**Comment by TanishMoral11** [comment]
Hi @Harshad09 , I’d like to work on this issue. Could you assign it to me?

**Comment by TanishMoral11** [comment]
hey @Harshad09 ,  if this issue is still open, can I work on this?

**Comment by therajanmaurya** [comment]
Moved on jira https://mifosforge.jira.com/browse/MW-201
